system/jhtree/jhtree.cpp

```diff
@@ -3381,7 +3384,11 @@ class CKeyMerger : public CKeyLevelManager
         if (activekeys>0) 
         {
             if (ctx)
+            {
                 ctx->noteStatistic(StNumIndexMerges, activekeys);
```

Comment:
@mckellyln is there any benefit in this being added to the complete line?  Would it help in any way?

Reply:
I think so - perhaps if its > Threshold then add to COMPLETE line.

Reply:
As far as I can tell from code inspection StNumIndexMerges is included in the accumulatedStatistics set and thus should be reported in the COMPLETE line already.